src/test/java/org/mskcc/cbio/portal/integrationTest/scripts/TestImportTabDelimData.java

```diff
@@ -509,5 +508,60 @@ private void addTestPatientAndSampleRecords(File file) throws FileNotFoundExcept
         }
         MySQLbulkLoader.flushAll();
     }
-    
+
+    @RunWith(SpringJUnit4ClassRunner.class)
+    @ContextConfiguration(locations = { "classpath:/applicationContext-dao.xml" })
+    @Rollback
+    @Transactional
+    public static class TestImportGenesetData {
+
+        @Test
+        public void testImportGenesetData() throws Exception {
+            ProgressMonitor.setConsoleMode(false);
+
+            // Open genesets test data file
+            File file = new File("src/test/resources/genesets/unit-test1_genesets.gmt");
```

Comment:
Please change to classpath import if you can

Reply:
@JREastonMarks Hi Jeremy, it's a nice improvement, but not related with my changes. Could you please create a ticket for this one to replace it in more places?

Reply:
@JREastonMarks I've just realised that the diff show this code as introduced in the current PR, but it's not the case. I am trying to understand why.

Reply:
@JREastonMarks I've just pushed a commit to take `TestImportGenesetData` class out of `TestImportTabDelimData` module. Sorry for the confusion. Not sure how it's happened.